internal/tsc_wrapped/tsc_wrapped.ts

```diff
@@ -139,15 +140,32 @@ function runOneBuild(
     return false;
   }
 
+  const emitter = bazelOpts.googmodule || bazelOpts.tsickle ? 'tsickle' : 'ts';
+  const emitResults: ts.EmitResult[] = [];
```

Comment:
maybe make this a `ts.EmitResult[]|tsickle.EmitResult[]`?

Reply:
Doesn't help much, still need to cast it below to narrow out the ts.EmitResult which is missing the `modulesManifest`